src/ElasticSearchIndexer.php

```diff
@@ -121,7 +121,12 @@ public function buildIndex(Iterator $documents): int
         if ($blockSize > 0) {
             $clientResponse = $this->client->bulk($params);
 
-            $this->logger->debug('client response: '. json_encode($clientResponse));
+            if ($clientResponse['errors'] === true) {
+                $errors = $this->checkErrors($clientResponse);
+                throw new ClientErrorResponseException($errors);
```

Comment:
You don't need a separate variable.
```suggestion
                throw new ClientErrorResponseException($this->checkErrors($clientResponse));
```